b2/console_tool.py

```diff
@@ -1360,10 +1360,35 @@ def _print_file_attribute(self, label, value):
         self._print((label + ':').ljust(20) + ' ' + value)
 
 
+class DownloadFileMixin(
+    ThreadsMixin,
+    ProgressMixin,
+    SourceSseMixin,
+    WriteBufferSizeMixin,
+    SkipHashVerificationMixin,
+    MaxDownloadStreamsMixin,
+    DownloadCommand,
+):
+    def get_download_info_from_local_filename(self, filename: str):
```

Comment:
```suggestion
    def _get_download_info_from_local_filename(self, filename: str):
```
the convention in this file is to prefix protected method names with a `_`